src/features/notification/assets/tabBarButtonGroup.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import React, {Component} from 'react';
+// import {View, Text} from 'react-native';
+import {ButtonGroup} from 'react-native-elements';
+
+type Props = {};
+
+type State = {
+  selectedIndex: number;
+};
+export default class tabBarButtonGroup extends Component<Props, State> {
```

Comment:
class or react component should be Capital in the first letter. Thus, this should be `TabBarButtonGroup`.